src/lib.rs

```diff
@@ -28,6 +28,7 @@
 //! assert_eq!(items, [(2, 20), (1, 10), (3, 30)]);
 //! ```
 
+#![forbid(missing_docs)]
```

Comment:
I think I accidentally typed `forbid`, but the difference is as follows:
> [forbid(C) is the same as deny(C), but also forbids changing the lint level afterwards](https://doc.rust-lang.org/reference.html#lint-check-attributes)

We can change to `deny` later, if need be.